addons/account/views/account_move_views.xml

```diff
@@ -722,14 +722,23 @@
                         <field name="state" widget="statusbar" statusbar_visible="draft,posted"/>
                     </header>
                     <div class="alert alert-warning mb-0" role="alert"
-                         invisible="state != 'draft' or not duplicated_ref_ids">
+                         invisible="state != 'draft' or not duplicated_ref_ids or move_type != 'in_invoice'">
                         Warning: this bill might be a duplicate of
-                        <button name="open_duplicated_ref_bill_view"
+                        <button name="open_duplicated_move_view"
                                 type="object"
                                 string="one of those bills"
                                 class="btn btn-link p-0"
                         />
                     </div>
+                    <div class="alert alert-warning mb-0" role="alert"
+                         invisible="state != 'draft' or not duplicated_ref_ids or move_type != 'out_invoice'">
+                        Warning: this invoice might be a duplicate of
+                        <button name="open_duplicated_move_view"
+                                type="object"
+                                string="one of those invoices"
+                                class="btn btn-link p-0"
+                        />
+                    </div>
```

Comment:
change `bill` for `document` and it's a oneline change without duplicating the whole stuff